src/utils/DateCalculate.js

```diff
@@ -0,0 +1,67 @@
+export function compareDateDifferences(rawDataTime) {
+  const currentTime = new Date();
+  const time = new Date(rawDataTime);
+
+  const timeDifference = currentTime - time; // 시간 차
```

Comment:
변수명으로 충분히 표현이 되기 때문에 이런 주석은 작성해주지 않는게 좋을 것 같아요.
```suggestion
  const timeDifference = currentTime - time;
```